src/main/js/html.js

```diff
@@ -731,19 +724,34 @@ var backgroundColorAdjustSuffix = "BackgroundColorAdjust";
         for (var i=0;i<lineList.length;i++) {
             var l = lineList[i].elements.length;
 
-            var se = lineList[i].elements[lineList[i].start_elem];
-
-            var ee = lineList[i].elements[lineList[i].end_elem];
-
             var pospadpxlen = Math.ceil(lp) + "px";
 
             var negpadpxlen = "-" + Math.ceil(lp) + "px";
 
             if (l !== 0) {
 
+                var se = lineList[i].elements[lineList[i].start_elem];
+
+                var ee = lineList[i].elements[lineList[i].end_elem];
+
+                if (se === ee) {
+
+                    // Check to see if there's any background at all
+                    elementBoundingRect = se.node.getBoundingClientRect();
```

Comment:
This doesn't work – elementBoundingRect doesn't have a var – but it's like this upstream, so I don't know how this fixes anything, unless the exception fixing it by stopping it going further when `se === ee`?

Reply:
Hmm, might be my bad habit of not declaring the var. It's been a while but I'm pretty sure this code fixed a test case at some point.

The intent of the code is to avoid adding line padding background in the inline direction to a line that contains no content, and therefore has no background area to extend. It checks for either width = 0 or height = 0 rather than jumping through other hoops trying to work out the writing direction.

Reply:
Fixed in e671704.